src/zenml/pipelines/pipeline_decorator.py

```diff
@@ -80,6 +87,15 @@ def pipeline(
         enable_artifact_metadata: Whether to enable artifact metadata or not.
         settings: Settings for this pipeline.
         extra: Extra configurations for this pipeline.
+        on_failure: Callback function in event of failure of the step. Can be
+            a function with three possible parameters,
+            `StepContext`, `BaseParameters`, and `Exception`,
+            or a source path to a function of the same specifications
+            (e.g. `module.my_file.my_function`).
```

Comment:
Tiny nit, but I would suggest removing the `my_file` part of the path for all of these, as file==module in this case